docs/topics/logging.txt

```diff
@@ -261,12 +265,37 @@ First, here's a simple configuration which writes all request logging from the
 If you use this example, be sure to change the ``'filename'`` path to a
 location that's writable by the user that's running the Django application.
 
-Second, here's an example of a fairly complex logging setup, configured using
+Second, here's an example of how to make the logging system print all of
+Django's logging to the console based on the value of an environment variable.
+It overrides the fact that ``django.request`` and ``django.security`` don't
```

Comment:
I would maybe add "by default" here: "It overrides the default logging configuration, where ``django.request`` and ``django.security`` loggers don't propagate their log entries."